src/app/service/categories/ServiceCategories.jsx

```diff
@@ -61,26 +69,32 @@ class ServiceCategories extends Component {
             <Header>What programs and services are available at this location?</Header>
           </div>
           <Accordion>
-            <Accordion.Item
-              active={active === 0}
-              onClick={() => this.onToggleOpen(0)}
-              title="All Services"
-              icon="home"
-            />
-            <Accordion.Content active={active === 0}>
-              <Selector fluid>
-                {taxonomy.map(category => (
-                  <Selector.Option
-                    key={category.id}
-                    onClick={() => this.onSelect(category)}
-                    active={selected[category.id]}
-                  >
-                    {category.name}
-                  </Selector.Option>
-                ))}
-                <Selector.Option align="center">+ Add another shelter service</Selector.Option>
-              </Selector>
-            </Accordion.Content>
+            {taxonomy.map((category, i) => (
+              <div key={category.id}>
+                <Accordion.Item
+                  active={active === i}
+                  onClick={() => this.onToggleOpen(i)}
+                  title={category.name}
+                  icon={iconNames[category.name]}
+                />
+                <Accordion.Content active={active === i}>
+                  <Selector fluid>
+                    {category.children.map(service => (
```

Comment:
The name "service" might be a bit confusing here (the actual "service" object is separate, and is what the next screens are about), I'd go with something like "subcategory".
Then again, our terminology in the team outside Tech often does refer to these subcategories as "services", so maybe this is fine. I dunno.